src/lv_core/lv_style.c

```diff
@@ -36,6 +36,12 @@
  **********************/
 LV_ATTRIBUTE_FAST_MEM static inline int32_t get_property_index(const lv_style_t * style, lv_style_property_t prop);
 static lv_style_t * get_alloc_local_style(lv_style_list_t * list);
+static void style_resize(lv_style_t *style, size_t sz);
+static lv_style_property_t get_style_prop(const lv_style_t *style, size_t idx);
+static uint8_t get_style_prop_id(const lv_style_t *style, size_t idx);
+static uint8_t get_style_prop_attr(const lv_style_t *style, size_t idx);
+static size_t get_prop_size(uint8_t prop_id);
+static size_t get_next_prop_index(uint8_t prop_id, size_t id);
```

Comment:
Can these be `inline`? This is the slowest part of the library so we should try not to lose any performance if possible.

Reply:
Do you mean transform in macros? Something like:

```c
#define GET_STYLE_PROP(s, i, prop) { uint8_t *__p= &prop; __p[0] = style->map[i]; __p[1] = style->map[i+1]; }
```

And so on...

I can do that or something else you are thinking but I think that the compiler inlines static functions if you use optimization properly and code keeps more clear... In my experience this kind of optimization usually is done pretty well by the compiler.

Anyways if, even considering that the compiler could inline those functions, you think it's better to inline the code manually then please let me know if you mean by using macros or expanding every occurrence.

Thanks.

Reply:
Adding the `inline` qualifier to the function should be enough.

Reply:
Ok. Thanks!

Reply:
Done. I retested in my 3 test platforms and it's ok.